src/components/snapshot/overview/SnapshotOverviewPanel.test.tsx

```diff
@@ -14,26 +14,64 @@ const duosFirecloudGroup = {
   lastSynced: '2022-04-04T18:53:45.158566Z',
 };
 
-const snapshot = {
-  id: 'uuid',
-  name: 'Test snapshot',
-  description: 'Test description',
-  createdDate: '2022-04-04T18:53:45.158566Z',
-  source: [
-    {
-      dataset: {
-        id: 'datasetId',
-        name: 'SourceDataset',
-        storage: [
-          { cloudResource: 'bigquery', region: 'us-east4' },
-          { cloudResource: 'firestore', region: 'us-east4' },
-          { cloudResource: 'bucket', region: 'us-east4' },
-        ],
+function createSnapshot(_duosFirecloudGroup?: any): any {
```

Comment:
I don't know if you want the `_` here right? I think that this indicates that it's an ignored field but you're using it below.  Also, I think there's a `DuosFirecloudGroupModel` type...not sure if it's the same as what's passed in here

Reply:
Oh, good points… I put the underscore because I had a naming conflict with a higher-level `duosFirecloudGroup` object, but I forgot that it is meant to represent an ignored field.  At any rate the conflict probably points to a code smell.

And yes, I think I can use that type here!  I might be able to use more types throughout, now that you mention it.